src/components/OIO/index.js

```diff
@@ -1,7 +1,9 @@
 import React, { Component } from 'react'
 import classNames from 'classnames'
+import defaults from './defaults'
 import style from './style.less'
 
+// TODO: Deprecate fontFamily and primaryColor props
```

Comment:
@asabhaney just would prefer to do that in a separate PR since lots of components use that primaryColor prop.